presto-main/src/main/java/com/facebook/presto/operator/HttpPageBufferClient.java

```diff
@@ -322,7 +322,7 @@ public void onSuccess(PagesResponse result)
 
                         if (!isNullOrEmpty(taskInstanceId) && !result.getTaskInstanceId().equals(taskInstanceId)) {
                             // TODO: update error message
-                            throw new PrestoException(REMOTE_TASK_MISMATCH, format("%s (%s)", REMOTE_TASK_MISMATCH_ERROR, HostAddress.fromUri(uri)));
+                            throw new PrestoException(REMOTE_TASK_MISMATCH, format("%s (%s)", REMOTE_TASK_MISMATCH_ERROR, fromUri(uri)));
```

Comment:
@raghavsethi  As we discussed, this is not a `PrestoTransportException`. The URI is used for exception message. And it get changed because we refactor to static import `fromUri` :)